src/connection/passwordless/actions.js

```diff
@@ -122,9 +142,10 @@ function sendSMSSuccess(id) {
 
 function sendSMSError(id, error) {
   const m = read(getEntity, 'lock', id);
-  const errorMessage = getErrorMessage(m, error);
+  const errorMessage = getErrorMessage(m, id, error);
   l.emitAuthorizationErrorEvent(m, error);
-  return swap(updateEntity, 'lock', id, l.setSubmitting, false, errorMessage);
+  swap(updateEntity, 'lock', id, l.setSubmitting, false, errorMessage);
```

Comment:
I didn't look at this in detail but this previous returned the value from `swap()`, does the fact that this function returns void now present any issues?

Reply:
We should be ok in this case, the only caller to `sendSMSError` discards the return value

https://github.com/auth0/lock/blob/872f7f767ce02cabf753f3e0672fe3fcc3602fa8/src/connection/passwordless/actions.js#L107